Cyral/Cyral.json

```diff
@@ -0,0 +1,1889 @@
+                    "timeRange": null,
+                    "coloringRules": null,
+                    "linkedDashboards": []
+                },
+                {
+                    "id": null,
+                    "key": "panel5F145873BF9CF949",
+                    "title": "Modifications",
+                    "visualSettings": "{\"general\":{\"mode\":\"distribution\",\"type\":\"table\"},\"series\":{}}",
+                    "keepVisualSettingsConsistentWithParent": true,
+                    "panelType": "SumoSearchPanel",
+                    "queries": [
+                        {
+                            "queryString": "($$logsrc )\n| json \"identity.group\",\"identity.endUser\",\"request.statementType\",\"response.records\",\"repo.name\" nodrop\n| where %\"identity.group\" matches \"{{Group}}\" and %\"identity.endUser\" matches \"{{endUser}}\" and (%\"request.statementType\"=\"DELETE\" or %\"request.statementType\"=\"INSERT\" or %\"request.statementType\"=\"UPDATE\")\n| json \"request.datasetsAccessed[*].accessType\" as accessType nodrop\n| json \"request.datasetsAccessed[*].dataset\" as dataSet nodrop\n| sum(%\"response.records\") as TotalRecords, count(accessType) as QueryTypes by %\"identity.endUser\", %\"repo.name\", dataSet",
+                            "queryType": "Logs",
+                            "queryKey": "A",
+                            "metricsQueryMode": null,
+                            "metricsQueryData": null,
+                            "tracesQueryData": null,
+                            "parseMode": "Manual",
+                            "timeSource": "Message"
+                        }
+                    ],
+                    "description": "",
+                    "timeRange": null,
+                    "coloringRules": null,
+                    "linkedDashboards": []
+                },
+                {
+                    "id": null,
+                    "key": "panel59636686B7F4FA4D",
+                    "title": "Top {{topKLimit}} Policy Violations",
+                    "visualSettings": "{\"general\":{\"mode\":\"distribution\",\"type\":\"table\"},\"series\":{}}",
+                    "keepVisualSettingsConsistentWithParent": true,
+                    "panelType": "SumoSearchPanel",
+                    "queries": [
+                        {
+                            "queryString": "($$logsrc)\n| json \"policyViolations[*].policyName\" as policyName nodrop\n| json \"policyViolations[*].severity\" as policySeverity nodrop\n| json \"policyViolations[*].accessType\" as policyAccessType nodrop\n| json \"identity.group\",\"identity.endUser\",\"request.statementType\" nodrop\n| where %\"identity.group\" matches \"{{Group}}\" and %\"identity.endUser\" matches \"{{endUser}}\" and %\"request.statementType\" matches \"{{statementType}}\"\n| count as Violations by policyName, policySeverity, policyAccessType\n| sort by Violations\n| limit {{topKLimit}}",
```

Comment:
![image](https://user-images.githubusercontent.com/3620468/127678354-c75a8e26-9633-42f5-bcbd-410bca71b367.png)

Seems there are violations without policyName should we ignore them or use "Unknown" in policyName?

Reply:
Good catch! This is a bug because I needed an extra file in the where clause of %"policyViolated"="true"